packages/client/src/lib/utils.ts

```diff
@@ -0,0 +1,7 @@
+import { clsx, type ClassValue } from 'clsx';
+import { twMerge } from 'tailwind-merge';
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
instead of turning off eslint, add return type to the function - I think it's string in this case